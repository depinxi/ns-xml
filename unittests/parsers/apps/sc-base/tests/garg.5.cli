values --argument-b 'And its value' -- "This should be ok"