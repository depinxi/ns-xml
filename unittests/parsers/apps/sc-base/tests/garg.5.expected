CLI: "values", "--argument-b", "And its value", "--", "This should be ok"
Value count: 1
Values: "This should be ok"
Error count: 0
Subcommand: values
switchA=False
argumentB=And its value
switchB=False
